.github/workflows/ci.yml

```diff
@@ -93,9 +93,9 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           # Run Error Prone on one module with a retry to ensure all runtime dependencies are fetched
-          $RETRY $MAVEN ${MAVEN_TEST} -T C1 clean test-compile -P gib,errorprone-compiler -pl ':trino-spi'
+          $RETRY $MAVEN ${MAVEN_TEST} -T C1 clean install -DskipTests -P gib,errorprone-compiler -pl ':trino-spi'
```

Comment:
Doesn't this make the assumption that `trino-spi` will never have errorprone issues and hence not go into a loop?

Reply:
it may have issues; in such case we will be retrying, but not for ever